### PR TITLE
cmd/scollector: negative -f filters and total_time metric for httpunit collector 

### PR DESCRIPTION
**Pull request by gbrayut**
Updated version of https://github.com/bosun-monitor/bosun/pull/1263 rebased on current master and with more documentation

### CODE REVIEW

File: cmd/scollector/collectors/httpunit.go [hunk 1]
```diff
@@ -10,33 +10,34 @@ import (
...
-	HTTPUnitPlans(filename, &httpunit.Plans{
-		Plans: plans,
-	})
+	HTTPUnitPlans(filename, &httpunit.Plans{Plans: plans}, freq)
 	return nil
 }
 
-func HTTPUnitPlans(name string, plans *httpunit.Plans) {
+func HTTPUnitPlans(name string, plans *httpunit.Plans, freq time.Duration) {
+	if freq < time.Second {
```
Comment (captncraig) [inline cmd/scollector/collectors/httpunit.go]: this seems a bit odd to me. If they didn't set it, and it was zero, I would understand. But if they set `15` expecting it to be seconds or minutes or who knows what, 5 mins is an odd default. Maybe error with a helpful message?

File: cmd/scollector/main.go [hunk 2]
```diff
@@ -124,11 +125,12 @@ func main() {
 		check(collectors.AddProcessDotNetConfig(p))
 	}
 	for _, h := range conf.HTTPUnit {
+		freq := time.Second * time.Duration(h.Freq)
```
Comment (captncraig) [inline cmd/scollector/main.go]: We use `time.ParseDuration` in some places, but apparently not others. Is a number really better than a duration string if you have to look up the docs to know it is seconds?

**Comment by captncraig** [comment]
Other than craig being confused about timings, it looks good to me.

**Comment by gbrayut** [comment]
@captncraig I changed it to use a duration string... will be clearer in the config files

**Pull request merged**